sdk/communication/communication-short-codes/swagger/README.md

```diff
@@ -5,9 +5,9 @@
 ## Configuration
 
 ```yaml
-package-name: azure-communication-short-codes
+package-name: "@azure/communication-short-codes"
```

Comment:
is this a recommended way to do specify the package name? should we update it in other packages too?

Reply:
Yes, this is feedback that I received in the PR for migrating the phone-numbers package: https://github.com/Azure/azure-sdk-for-js/pull/20633#discussion_r821033569

This changes the user agent used by the generated client; it goes from `azsdk-js-azure-communication-short-codes/1.0.0-beta.2` to `azsdk-js-communication-short-codes/1.0.0-beta.2`, which was consistent with the user agent prior to the migration.

Reply:
cool, will do the same change in the identity package